backend/fms_core/services/sample.py

```diff
@@ -37,7 +37,7 @@ def create_full_sample(name, volume, collection_site, creation_date,
         biosample_data = dict(
             collection_site=collection_site,
             **(dict(individual=individual) if individual is not None else dict()),
-            **(dict(alias=alias) if alias is not None else dict()),
+            **(dict(alias=alias) if alias else dict(alias=name)),
```

Comment:
you should not get rid of the is not None i think. If someone submit a falsy value (e.g. numeric 0) we'll get mixups.